frontend/src/app/tasks/tasks.component.html

```diff
@@ -15,7 +15,8 @@
       </div>
       <div class="task-buttons" *ngIf="task.model.status === 'TODO'">
         <button (click)="edit(task, $event)" class="btn btn-secondary btn-sm edit-button"><span class="fa fa-edit"></span></button>
-        <button (click)="assign(task, $event)" class="btn btn-outline-primary btn-sm assign-button" ><span class="fa fa-hand-o-right"></span></button>
+        <button (click)="assign(task, $event)" class="btn btn-outline-primary btn-sm assign-button" *ngIf="!task.model.assignee"><span class="fa fa-hand-o-right"></span></button>
+        <button (click)="unassign(task, $event)" class="btn btn-outline-secondary btn-sm unassign-button" *ngIf="task.model.assignee"><span class="fa fa-hand-o-left"></span></button>
```

Comment:
I'd rather use this icon: http://fontawesome.io/icon/hand-paper-o/: it's more different than the one used to assign, and can be seen as a "no, stop, I don't want to do this" sign.